modules/exploits/multi/ssh/sshexec.rb

```diff
@@ -77,7 +120,7 @@ def initialize
       [
         OptString.new('USERNAME', [ true, "The user to authenticate as.", 'root' ]),
         OptString.new('PASSWORD', [ true, "The password to authenticate with.", '' ]),
-        OptString.new('RHOST', [ true, "The target address" ]),
+        Opt::RHOST('RHOST', [ true, "The target address" ]),
```

Comment:
:+1:

Reply:
You should do `Opt::RHOST`, though. I don't think you want the default address to be the string `RHOST`.

https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/opt.rb#L44

Reply:
Ohh; yeah.... duh.  Thanks!